src/extension/netcdf-out/src/main/java/org/geoserver/netcdfout/NetCDFOutStatus.java

```diff
@@ -50,10 +51,33 @@ public boolean isEnabled() {
         return true;
     }
 
+    public String Nc4Version() {
+        try {
+                //This reflection code is to deal with JNA being an optional jar.
+                //Its the same as;
+                //     Nc4prototypes nc4 = (Nc4prototypes) Native.loadLibrary("netcdf", Nc4prototypes.class);
+                //     return nc4.nc_inq_libvers();
+            Class com_sun_jan_Native = Class.forName("com.sun.jna.Native");          
```

Comment:
Not sure what happened - git wasn't picking up my changes.  
Should be correct now.